client/btc.go

```diff
@@ -107,3 +107,52 @@ func (cli *Client) SignSegWit(roleId, pubkey string, request *v0.BtcSignRequest,
 	}
 	return &decoded, "", nil
 }
+
+func (cli *Client) SignPsbt(roleId, pubkey string, request *v0.PsbtSignRequest, mfaId, mfaConfirmation *string) (*v0.PsbtSign200Response, string, error) {
+	authResp, err := cli.CreateRoleToken(&v0.CreateTokenRequest{
+		Purpose: "sign psbt",
+		Scopes:  []api.Scope{api.SIGNBTCPSBT},
```

Comment:
- ack that you're following the convention above
- this and other bits should probably be refactored to not need management scopes